chezbetty/locale/fa/LC_MESSAGES/betty.po

```diff
@@ -0,0 +1,736 @@
+#: chezbetty/templates/user_info.jinja2:45
+#: chezbetty/templates/deposit_complete.jinja2:12
+msgid "Deposit"
+msgstr "واریز"
+
+#: chezbetty/templates/deposit.jinja2:21
+msgid "Cash"
+msgstr "پول نقد"
+
+#: chezbetty/templates/deposit.jinja2:22
+msgid "Your Deposit Limit: %(limit)s"
+msgstr "محدودیت واریز‫:‬ %(limit)"
+
+#: chezbetty/templates/deposit.jinja2:45
+msgid "Bitcoin"
+msgstr "بیت‌کوین"
+
+#: chezbetty/templates/deposit.jinja2:56
+msgid "Credit Card"
+msgstr "کارت‌اعتباری"
+
+#: chezbetty/templates/deposit.jinja2:58 chezbetty/templates/index.jinja2:36
+msgid "Betty now accepts credit cards!"
+msgstr "امکان پذیرش کارت اعتباری اضافه شد‫!‬"
+
+#: chezbetty/templates/deposit.jinja2:61 chezbetty/templates/index.jinja2:38
+msgid ""
+"To deposit using a credit card, please visit chezbetty.eecs.umich.edu/user "
+"in your browser."
+msgstr ""
+"برای واریز پول از طریق کارت اعتباری، لطفا در مرورگر خود به آدرس "
+"chezbetty.eecs.umich.edu/user رجوع کنید."
+
+#: chezbetty/templates/purchase.jinja2:3 chezbetty/templates/purchase.jinja2:4
+#: chezbetty/templates/user_info.jinja2:39
+msgid "Purchase"
+msgstr "خرید"
+
+#: chezbetty/templates/purchase.jinja2:17
+#: chezbetty/templates/purchase_complete.jinja2:13
+msgid "Order Summary"
+msgstr "خلاصه سفارش"
+
+#: chezbetty/templates/purchase.jinja2:34 chezbetty/templates/user.jinja2:45
+#: chezbetty/templates/items.jinja2:36 chezbetty/templates/items.jinja2:61
+#: chezbetty/templates/items.jinja2:84
+#: chezbetty/templates/purchase_complete.jinja2:20
+msgid "Item"
+msgstr "جنس"
+
+#: chezbetty/templates/purchase.jinja2:35 chezbetty/templates/user.jinja2:46
+#: chezbetty/templates/purchase_complete.jinja2:21
+msgid "Quantity"
+msgstr "تعداد"
+
+#: chezbetty/templates/purchase.jinja2:36 chezbetty/templates/user.jinja2:47
+#: chezbetty/templates/purchase_complete.jinja2:22
+msgid "Item Price"
+msgstr "قیمت جنس"
+
+#: chezbetty/templates/purchase.jinja2:37 chezbetty/templates/user.jinja2:48
+#: chezbetty/templates/purchase_complete.jinja2:23
+msgid "Total Price"
+msgstr "قیمت کل"
+
+#: chezbetty/templates/purchase.jinja2:47
+msgid "Order Empty"
+msgstr "سبد شما خالی‌است"
+
+#: chezbetty/templates/purchase.jinja2:48
+msgid "Scan an item to begin"
+msgstr "برای شروع، جنس اول را اسکن کنید‫.‬"
+
+#: chezbetty/templates/purchase.jinja2:57 chezbetty/templates/user.jinja2:67
+#: chezbetty/templates/purchase_complete.jinja2:42
+msgid "Subtotal"
+msgstr "جمع جزء"
+
+#: chezbetty/templates/purchase.jinja2:61
+msgid "Good Standing Discount"
+msgstr "تخفیف خوش‌حسابی"
+
+#: chezbetty/templates/purchase.jinja2:67 chezbetty/templates/user.jinja2:76
+#: chezbetty/templates/purchase_complete.jinja2:53
+msgid "Total"
+msgstr "مجموع کل"
+
+#: chezbetty/templates/purchase.jinja2:78
+msgid "Pay With Your User Account"
+msgstr "با حساب کاربری خود پرداخت کنید"
+
+#: chezbetty/templates/purchase.jinja2:80
+msgid "Pay With %(pool_name)s Pool"
+msgstr "با حساب مشترک %(pool_name) پرداخت کنید‫.‬"
```

Comment:
I'm guessing these are issues for all of the strings, I won't comment on all of them